neqo-crypto/src/err.rs

```diff
@@ -34,7 +29,7 @@ pub mod nspr {
 pub type Res<T> = Result<T, Error>;
 
 #[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq)]
-#[allow(clippy::pub_enum_variant_names)]
+#[allow(renamed_and_removed_lints, clippy::pub_enum_variant_names)] // until we require rust 1.54
```

Comment:
This will need another fix once we get to 1.54.  Maybe instead:

```suggestion
#[allow(renamed_and_removed_lints, clippy::pub_enum_variant_names)] // rust 1.54 will require a different approach
```